File: Python Code/Daily_Flash_Code_0/Week 3/27Jan/12-DailyFlash_Solutions/12-DailyFlash_Solutions/27_Jan_Solutions_One/Java/RangeOfEven.java
import java.io.*;

class RangeOfEven{

	
	public static void main(String args[])throws IOException{

		BufferedReader br =new BufferedReader(new InputStreamReader(System.in));

		System.out.printf("Min of Series : ");
		int lowerlimit = Integer.parseInt(br.readLine());
		System.out.printf("Max of Series : ");
		int upperlimit = Integer.parseInt(br.readLine());


		for(int itr = lowerlimit ; itr < upperlimit ; itr++){

			System.out.print(itr%2==0?itr:" ");
	
		}	

	}	
}	

File: Python Code/Daily_Flash_Code_0/Week 3/27Jan/12-DailyFlash_Solutions/12-DailyFlash_Solutions/27_Jan_Solutions_One/Cpp/RangeOfEven.cpp
#include<iostream>

int main(){

	
	int lowerlimit;
	int upperlimit;


	printf("Min of Series : ");
	scanf("%d",&lowerlimit);

	printf("Max of Series : ");
	scanf("%d",&upperlimit);

	printf("Series Of Even number's ranging from %d & %d is : ",lowerlimit,upperlimit);
	for(int itr = lowerlimit ; itr < upperlimit ; itr++){
		if(itr%2==0)
			printf("%d ",itr);	

	}	

}		
